bin/reth/Cargo.toml

```diff
@@ -56,6 +56,7 @@ reth-nippy-jar.workspace = true
 
 # crypto
 alloy-rlp.workspace = true
+alloy-chains.workspace = true
```

Comment:
this is not needed here

Reply:
but I am using alloy-chains in bin/reth, if I remove this, I am unable to use alloy_chains::Chain?
How to proceed here?

Reply:
these additional deps are not required, because `Chain` is still reexported:

https://github.com/paradigmxyz/reth/blob/main/crates/primitives/src/lib.rs#L54-L54

so none of this should be required.